tests/test_app.py

```diff
@@ -53,14 +53,15 @@ def tearDown(self):
         connection_manager.rollback()
         connection_manager.put()
 
-    
+    @classmethod
     def tearDownClass(self):
         # Delete the content of the storage account
         FERTISCAN_STORAGE_URL = os.getenv("FERTISCAN_STORAGE_URL")
         with ContainerClient.from_connection_string(
             FERTISCAN_STORAGE_URL, container_name=f"user-{self.username}"
```

Comment:
It takes the user id instead of the username. You can save the user id from the `/signup` response in `setUp`.